addons/website_blog/static/src/scss/primary_variables.scss

```diff
@@ -1,3 +1,2 @@
 
-@mixin s-latest-posts-js-get-posts-hook {};
```

Comment:
Move this as 000_variables.scss like I did for some other snippets.

Reply:
How would versioning work for those? Am I missing something?

Reply:
Actually this entire file is no longer needed because of the move.